wercker.yml

```diff
@@ -5,11 +5,14 @@ services:
 build:
     steps:
         - virtualenv:
-            install_wheel: false
+            install_wheel: true
         - script:
             name: Install remodel
             code: pip install .
         - script:
+            name: Install python-dateutil (for six)
```

Comment:
That is what I was thinking also; I guess the `six` dependency in `setup.py` gets it going. I don't like "polluting" the requirements, but unfortunately this is the only way to support both Python 2 and 3.